tests/test_account_activation.py

```diff
@@ -65,9 +59,6 @@ def test_activate_account_valid_token_specified(self, mock_object):
         response = self.app.get('/register/activate-account/' + token, headers=self.headers)
 
         self.assertEqual(response.status_code, 302)
-        self.assertTrue(
-            bytes('You should be redirected automatically to target URL', encoding='UTF-8') in response.data)
-        self.assertTrue('/sign-in/?account_activated=True'.encode() in response.data)
 
     # ============== YOUR LINK HAS EXPIRED PAGE ===============
```

Comment:
Do we want to keep these? we don't have them in the other test files I don't think